### PR TITLE
Code Cleaning: Format to google style and fix errors from linting results

### PR DESCRIPTION
**Pull request by chrishkchris**
I have formatted the python code according to google style using yapf 
https://github.com/nusdbsystem/singa-easy/wiki/Development-Guide

### CODE REVIEW

**Comment by chrishkchris** [comment]
Ready for review/merge

**Comment by chrishkchris** [comment]
Combined all the commits into 2 commits. Ready for review/merge.

File: singa_easy/modules/explanations/maskmap/misc_functions.py [hunk 1]
```diff
@@ -203,7 +209,7 @@ def recreate_image(im_as_var):
         recreated_im (numpy arr): Recreated image in array
     """
     reverse_mean = [-0.485, -0.456, -0.406]
-    reverse_std = [1/0.229, 1/0.224, 1/0.225]
+    reverse_std = [1 / 0.229, 1 / 0.224, 1 / 0.225]
```
Comment (nudles) [inline singa_easy/modules/explanations/maskmap/misc_functions.py:212]: is this required by the pylint? or done by the autoformat tool?
1/0.229 is faster to code:-)

Comment (chrishkchris) [reply singa_easy/modules/explanations/maskmap/misc_functions.py:212]: this is done by the autoformat tool, it adds a space

**Pull request merged**